emoji-substitution/emojiMap.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file contains the Map of word --> emoji substitutions.
+ */
+
+let dictionary = new Map();
+dictionary.set('apple', '🍎');
+dictionary.set('banana', '🍌');
+dictionary.set('bang', '💥');
+dictionary.set('baseball', '⚾');
+dictionary.set('basketball', '🏀');
+dictionary.set('beer', '🍺');
+dictionary.set('bicycle', '🚴');
+dictionary.set('bike', '🚴');
+dictionary.set('bomb', '💣');
+dictionary.set('boy', '👦');
+dictionary.set('bug', '🐛');
+dictionary.set('burger', '🍔');
+dictionary.set('burn', '🔥');
+dictionary.set('cake', '🎂');
+dictionary.set('candy', '🍬');
+dictionary.set('cat', '🐱');
+dictionary.set('celebration', '🎉');
+dictionary.set('cheeseburger', '🍔');
+dictionary.set('cookie', '🍪');
+dictionary.set('cool', '😎');
+dictionary.set('cry', '😢');
+dictionary.set('dog', '🐶');
+dictionary.set('doge', '🐕');
+dictionary.set('earth', '🌎');
+dictionary.set('explode', '💥');
+dictionary.set('fart', '💨');
+dictionary.set('fast', '💨');
+dictionary.set('female', '👩');
+dictionary.set('fire', '🔥');
+dictionary.set('fish', '🐟');
+dictionary.set('flame', '🔥');
+dictionary.set('flower', '🌹');
+dictionary.set('food', '🍕');
+dictionary.set('football', '🏈');
+dictionary.set('girl', '👧');
+dictionary.set('golf', '⛳');
+dictionary.set('hamburger', '🍔');
+dictionary.set('happy', '😀');
+dictionary.set('horse', '🐴');
+dictionary.set('hot', '🔥');
+dictionary.set('kiss', '😘');
+dictionary.set('laugh', '😂');
+dictionary.set('lit', '🔥');
+dictionary.set('lock', '🔒');
+dictionary.set('lol', '😂');
+dictionary.set('love', '😍');
+dictionary.set('male', '👨');
+dictionary.set('man', '👨');
+dictionary.set('monkey', '🐵');
+dictionary.set('moon', '🌙');
+dictionary.set('note', '📝');
+dictionary.set('paint', '🎨');
+dictionary.set('panda', '🐼');
+dictionary.set('party', '🎉');
+dictionary.set('pig', '🐷');
+dictionary.set('pizza', '🍕');
+dictionary.set('planet', '🌎');
+dictionary.set('rose', '🌹');
+dictionary.set('sad', '😢');
+dictionary.set('sleep', '😴');
+dictionary.set('smile', '😀');
+dictionary.set('smiley', '😀');
+dictionary.set('soccer', '⚽');
+dictionary.set('star', '⭐');
+dictionary.set('sun', '☀️');
+dictionary.set('sunglasses', '😎');
+dictionary.set('surprised', '😮');
+dictionary.set('tree', '🌲');
+dictionary.set('trophy', '🏆');
+dictionary.set('win', '🏆');
+dictionary.set('wind', '💨');
+dictionary.set('wine', '🍷');
+dictionary.set('wink', '😉');
+dictionary.set('woman', '👩');
```

Comment:
One thing I just thought of is `substitution.js` could sort these by length at the very beginning so even if some words are subsets of others (as is the case with "man" and "woman"), "woman" would be applied first.

This way it wouldn't matter what order they appear in `emojiMap.js`.